compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -956,6 +956,12 @@ error.duplicate.key.in.table.literal=\
 error.duplicate.key.in.record.literal.spread.op=\
   invalid usage of {0} literal: duplicate key ''{1}'' via spread operator ''{2}''
 
+error.inclusive.record.literal.with.spread.op=\
+  invalid usage of {0} literal: inclusive record ''{1}'' with spread operator is not allowed
```

Comment:
Let's fix this also for mapping constructor expression and maps. And IMO we should say "spread field."

Reply:
I think this error is not descriptive enough. We should also say why not allowed or a suggestion to use `never`. 
Currently, this may give an idea that in general, we cannot use inclusive records with spread operator.